openshift/examples/v1_namespace.yml

```diff
@@ -0,0 +1,40 @@
+# v1_namespace.yml
```

Comment:
I like how the test data and validations are broken apart from the test implementation. Not too sure about the exact representation. I think that is something that we could evolve over time, though and not something that we should bikeshed over right now.